command/agent/alloc_endpoint.go

```diff
@@ -520,7 +520,7 @@ func (s *HTTPServer) allocExec(allocID string, resp http.ResponseWriter, req *ht
 
 // readWsHandshake reads the websocket handshake message and sets
 // query authentication token, if request requires a handshake
-func readWsHandshake(readFn func(interface{}) error, req *http.Request, q *structs.QueryOptions) error {
+func (s *HTTPServer) readWsHandshake(readFn func(interface{}) error, req *http.Request, q *structs.QueryOptions) error {
```

Comment:
Now that you've removed your debugging logging, you don't need this change to make this function a method anymore. That also fixes the linter error you're getting on `command/agent/alloc_endpoint_test.go`